components/alert/alert.css

```diff
@@ -0,0 +1,82 @@
+.alert{
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 1.6rem;
+    padding: 1.5rem 2rem;
+    border-radius: 2px;
+    width: 100%;
+    max-width: 700px;
```

Comment:
try to use rem

Reply:
hey resolved the changes. I hope my color naming conventions are good. If you think they are ok please approve my changes.